docker-compose.yml

```diff
@@ -11,21 +11,6 @@ services:
       timeout: 3s
       retries: 5
 
-  childchain_db:
-    image: postgres
-    restart: always
-    environment:
-      POSTGRES_USER: omisego_dev
-      POSTGRES_PASSWORD: omisego_dev
-      POSTGRES_DB: omisego_dev
-    ports:
-      - "5432:5432"
```

Comment:
I was tempting to change `watcher_db` host port to std - but I thing it's actually better in the case when local postgres is running (and took std port)